app/views/layouts/_footer.html.erb

```diff
@@ -2,7 +2,7 @@
   <div id="footer-container" class="container <%= "centered-footer" unless current_page.include?("stories-show") %>">
     <div class="inner-footer-container">
       <a href="/">Home</a> <a href="/about">About</a> <a href="/privacy">Privacy Policy</a> <a href="/terms">Terms of Use</a> <a href="/contact">Contact</a> <a href="/code-of-conduct">Code of Conduct</a>
-      DEV Community copyright 2018 🔥
+      DEV Community copyright 2019 🔥
```

Comment:
What do you think about using a computed value? This way it will progress with the years:

```ruby
<%= Time.current.year %>
```

Reply:
Good call. @micmcgrorty do you want to make that update in both places?

Reply:
Oops. Just realized we already merged something. @benhalpern is there a reason why we've continued to hardcode the year?